### PR TITLE
Code Review for Feature1

### PR DESCRIPTION
**Pull request by lordbritishix**

### CODE REVIEW

File: src/AccessLog.java [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    private String operation;
+    private String url;
+    private String version;
+    private String response;
+    private String size;
+
+    private int tokens = 0;
+
+
+    public int parseLog(String log) throws ParseException {
```
Comment (lordbritishix) [inline src/AccessLog.java]: Mas mainam kung un AccessLog ay data class lang. Hiwalay ang tiga lagay ng data.

Comment (lordbritishix) [inline src/AccessLog.java]: AccessLog parseLog(String log)

File: src/AccessLog.java [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+        StringTokenizer stringTokenizer = new StringTokenizer(log);
+        tokens = stringTokenizer.countTokens();
+
+        source = stringTokenizer.nextToken();
+        stringTokenizer.nextToken();
```
Comment (lordbritishix) [inline src/AccessLog.java]: di mo ba kailangan yun result ng nextToken na ito?

File: src/AccessLog.java [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+        return tokens;
+    }
+
+
+    public int countToken(String log){
+
+        StringTokenizer stringTokenizer = new StringTokenizer(log);
+        tokens = stringTokenizer.countTokens();
+
+<<<<<<< HEAD
```
Comment (lordbritishix) [inline src/AccessLog.java]: merge conflict?

File: src/AccessLog.java [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+    public String splitOperation(String operation){
+        String[] tempOperation = StringUtils.splitString(operation,"  ");
+        operation = tempOperation[0];
+    return operation;
+    }
+
+
+
+    public String getSource() {
```
Comment (lordbritishix) [inline src/AccessLog.java]: Lombok

File: src/AccessLog.java [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+ */
+public class AccessLog {
+
+    private String source;
+    private String timeStamp;
+    private String operation;
+    private String url;
+    private String version;
+    private String response;
+    private String size;
```
Comment (lordbritishix) [inline src/AccessLog.java]: long dapat ang size

File: src/AccessLog.java [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+    }
+
+
+    public String formatDateUTC(String timeStamp) throws ParseException {
+        Date date = new Date();
+        DateFormat simpleDateFormat = new SimpleDateFormat("dd/MMM/yyyy:HH:mm:ss z");
+        String tempTime = "";
+        String[] tempTimeStamp = StringUtils.splitString(timeStamp,"[");
+
+            for (int x = 1; x <= tempTimeStamp.length-1; x++){
```
Comment (lordbritishix) [inline src/AccessLog.java]: pwede din line.substring(1, line.length() - 1);

File: src/AccessLog.java (hunk 1 above)
Comment (lordbritishix) [inline src/AccessLog.java]: Wala ka pang pang parse para sa error log

File: src/TestAccessLog.java [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+
+public class TestAccessLog {
+
+    @Test
+    public void testLine() throws ParseException {
+
+        AccessLog accessLog = new AccessLog();
+        String line = "64.242.88.10 - - [07/Mar/2004:14:05:49 -0800] GET /twiki/bin/edit/Main/Double_bounce_sender?topicparent=Main.ConfigurationVariables HTTP/1.1 401 12846";
+        int test = accessLog.parseLog(line);
```
Comment (lordbritishix) [inline src/TestAccessLog.java]: kulang pa ang assert mo, dapat sa lahat ng fields na pinarse

File: src/TestAccessLog.java [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+        Assert.assertEquals(10,test);
```
Comment (lordbritishix) [inline src/TestAccessLog.java]: pati mga negative at edge cases
